dbcon/joblist/tuplehashjoin.cpp

```diff
@@ -881,7 +883,7 @@ void TupleHashJoinStep::hjRunner()
   }
 
   // todo: forwardCPData needs to grab data from djs
-  if (!djs)
+  if (!djs.size())
```

Comment:
```suggestion
  if (djs.empty())
```
This form looks easier to read IMHO. What do you think?